Software/scan_key/AuthenticationSQL.py

```diff
@@ -1,42 +1,42 @@
 from on_off import *
 from ErrorSQL import *
 import os
+import subprocess
 
 def machineAuth(id, cursor):
-    os.system("hostname > tmp")
-    currMachineID = open('tmp', 'r').read()
-    currMachineID = currMachineID.split()
+    currMachineID = subprocess.check_output(['hostname'])
+    name = name.strip()
 
-    cursor.execute("select machineID, machineType from MACHINE")
+    cursor.execute("select machineID, machineType FROM MACHINE WHERE t1String = " + id)
 
-    data1 = cursor.fetchall()
+    data1 = cursor.fetchone()
 
     machineType = None
-
+    
     for origMachineID in data1:
-        if(origMachineID[0] == currMachineID[0]):
+        if(origMachineID[0] == currMachineID):
             machineType = origMachineID[1]
```

Comment:
Delete lines 14-18.

We just requested the machineType for our machine. Why go around your ass to get to your elbow? :)